app/views/report_templates/generate.html.erb

```diff
@@ -29,5 +29,8 @@
     <% end %>
   <% end %>
 
+  <%= checkbox_f f, :send_mail, label: _('Send report to e-mail') %>
+  <%= text_f     f, :mail_to, label: _('Mail(s) to deliver report to'), label_help: _('Values has to be valid e-mails delimited by ";"') %>
```

Comment:
I don't like the alignment here too much, label is not aligned either (3rd arg - the hash), in order to be consistent, could you please keep one space before `f` here?

Reply:
yeah, the idea of alignment dissapeared after few changes and doesn't make sense anyway, will remove it, thanks :)